src/python/doc/barycenter_user.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+Barycenter user manual
+================================
+Definition
+----------
+
+.. include:: barycenter_sum.inc
+
+This implementation is based on ideas from "Frechet means for distribution of persistence diagrams", Turner et al. 2014.
+
+Function
+--------
+.. autofunction:: gudhi.barycenter.lagrangian_barycenter
+
+
+Basic example
+-------------
+
+This example computes the Frechet mean (aka Wasserstein barycenter) between four persistence diagrams.
+It is initialized on the 4th diagram, which is the empty diagram. It is encoded by np.array([]).
```

Comment:
What is the significance of the initialization diagram? Does it only change the speed? Does it change the result? How can a user decide how to initialize?
I don't think you need to say that np.array([]) is empty.

Reply:
It actually has a lot of importance: the energy functional is not convex in the PD space so the output (and its quality) depends on the initialization. There is however no heuristic (afaik) to pick a good initial diagram. 

I should change the default initialization (which is currently "one of the dgm at random") as it might be counter intuitive. Maybe we could start with the empty dgm (which I think is a very bad choice actually...), or the "first dgm of the pdiagset", so that the user gets a deterministic output?

I will implement later another estimation procedure ("Eulerian" instead of "Lagrangian"), which will be convex this time (at the cost of optimizing on a regular grid, returning a "persistence surface" instead of an image).

Reply:
My questions meant that this requires more documentation, not particularly that the code needs changing. Offering the user an option to initialize differently is more useful if the user understands what that means.

Reply:
Improved the doc (hopefully).